zsh/.zshrc

```diff
@@ -129,6 +129,13 @@ _zqs-purge-setting() {
 
 # Convert the old settings files into new style settings
 function _zqs-update-stale-settings-files() {
+  # Convert .zqs-additional-plugins to new format
```

Comment:
Putting it here is better than my earlier suggestion.

Reply:
Did I put the conversion code in the wrong place?

Reply:
No, this is perfect.

Reply:
Great!